pkg/token/execCredentialWriterAPIVersion_test.go

```diff
@@ -0,0 +1,69 @@
+package token
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/Azure/go-autorest/autorest/adal"
+	"k8s.io/client-go/pkg/apis/clientauthentication"
+)
+
+
+func TestExecCredentialWriterAPIVersion() (t *testing.T) {
+	testData1 := []struct {
+		name               string
+		execInfoEnvTest    string
+		stringBufferTest   bytes.Buffer
```

Comment:
you don't need it in the test data, just create a local bytes.Buffer after L59

Reply:
besides, https://pkg.go.dev/bytes#NewBuffer, 
> In most cases, new(Buffer) (or just declaring a Buffer variable) is sufficient to initialize a Buffer.

Reply:
Addressed these two in the most recent commit 244a97c